### DISCUSSION TITLE
Is there a way to load and play at the same time with one click on Iphone Safari?

### DISCUSSION THREAD

**Discussion by nshen** [katspaugh/wavesurfer.js#2899] [Q&A]
Hi, thanks for the cool project. I am making a music player, current when user clicks an item on the playlist, I create wavesurfer and  call `load()` method to load the music, then when the `ready` event fires I call `play()` method to play the music.  It worked as expected until I tested it on iPhone Safari.

I found that on iPhone Safari, I have to click item to load() first, then click play button the second time to play normally. 

I think that might because iOS didn't allow me to programmatically call `play()` method in the ready event handler 

So Is there a way to load and play at the same time with only one click on Safari?

**Comment by katspaugh** [comment]
Perhaps try the `autoplay: true` option. I haven't tested it on iOS though.
